modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -1056,12 +1056,24 @@ protected int processStartTag() throws Exception {
 				LinkTag linkTag = new LinkTag();
 
 				linkTag.setCssClass(
-					"nav-btn nav-btn-monospaced btn btn-primary");
+					"d-md-none nav-btn nav-btn-monospaced btn btn-primary");
```

Comment:
No FF required on SSR side?

Reply:
🤦‍♀️ On it!

Reply:
Fixed in 🥲  https://github.com/liferay-frontend/liferay-portal/pull/1820/commits/4fbdee70b83b21e1faa4bae3d1b49c16dee772b3. I tried to generate the class using a ternary operator, but sourceFormat doesn't like it 🙄